charts/ccpay-bubble-frontend/values.yaml

```diff
@@ -20,7 +20,7 @@ nodejs:
 
     CCPAY_BUBBLE_MICROSERVICE: ccpay_bubble
     CCPAY_BUBBLE_URL:  https://ccpay-bubble-frontend-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal
-    PAYHUB_API_URL: http://payment-api-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal
+    PAYHUB_API_URL: http://payment-api-pr-737.service.core-compute-preview.internal
```

Comment:
I believe this change is required for PR testing but make sure we revert this for demo/master push

Reply:
Sure, I will remove this for demo/master push